File: app/src/main/java/com/aero51/moviedatabase/repository/model/tmdb/tvshow/TvShowSearchResult.java
package com.aero51.moviedatabase.repository.model.tmdb.tvshow;

import com.aero51.moviedatabase.repository.model.tmdb.movie.TopRatedMovie;

import java.util.List;

public class TvShowSearchResult {

    private Integer page;
    private Integer total_results;
    private Integer total_pages;

    private List<TvShow> results;

    public TvShowSearchResult(Integer page, Integer total_results, Integer total_pages, List<TvShow> results) {
        this.page = page;
        this.total_results = total_results;
        this.total_pages = total_pages;
        this.results = results;
    }

    public Integer getPage() {
        return page;
    }

    public Integer getTotal_results() {
        return total_results;
    }

    public Integer getTotal_pages() {
        return total_pages;
    }

    public List<TvShow> getResults() {
        return results;
    }
}

File: app/src/main/java/com/aero51/moviedatabase/repository/model/tmdb/tvshow/TvShowSearchResult.kt
package com.aero51.moviedatabase.repository.model.tmdb.tvshow

class TvShowSearchResult {
    var page: Int = 0
    var total_results: Int? = null
    var total_pages: Int? = null
    var results: List<TvShow>? = null

    class TvShow {
        var backdrop_path: String? = null
        var first_air_date: String? = null
        var genre_ids: List<Int>? = null
        var id: Int? = null
        var name: String? = null
        var origin_country: List<String>? = null
        var original_language: String? = null
        var original_name: String? = null
        var overview: String? = null
        var popularity: Double? = null
        var poster_path: String? = null
        var vote_average: Double? = null
        var vote_count: Int? = null
    }
}